scripts/utils/price-utils.js

```diff
@@ -139,9 +139,52 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return true
   }
 
+  const checkNoProfitableOffer = async (order, exchange, currentMarketPrice) => {
+    const tokenInfo = fetchTokenInfoFromExchange(exchange, [order.buyToken, order.sellToken])
+
+    if (currentMarketPrice === undefined) {
+      currentMarketPrice = await getDexagPrice(
+        (await tokenInfo[order.buyToken]).symbol,
+        (await tokenInfo[order.sellToken]).symbol
+      )
```

Comment:
I've noticed that very often when I pushed my commits to Github Travis told me that the tests failed just because the one call we make to dex.ag failed to retrieve the price for whatever reason, and I had to rerun Travis to make the tests pass. And this is just a single call to their servers.
As I read this PR, you call dex.ag for every order currently active for a bracket. Considering how many brackets we deployed so far (>100?) I'd say it's basically certain that one of these calls will fail during the execution of the script. Or we'd at least run into some rate limiting. I'd say that at least the price should be recovered only once for each pair, similar to what we do for ERC20 token info. This would at least reduce the number of calls to ~10, which hopefully doesn't make us rerun the verification scripts more than a few times to see if it works. Ideally we would also catch failures and retry after a while for a certain amount of times (let's say 5?).